.github/workflows/linter.yml

```diff
@@ -0,0 +1,65 @@
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+# For the Web Almanac we have the following rule changes from the defaults:
+#  * Markdown - we're a bit more relaxed about this since authors may be less technical
+#  * Python - Only change from default is to set ignored-modules=flask, flask-talisman...etc.
+#  * CSS - Tweaked to ignore some Prince/Normalise exceptions and some whitespace exceptions
+#  * JS - default rules
+#  * JSON - default rules
+#  * YML - default rules
+# Exception config files are in the .github/linters directory
+
+######################################
+# Start the job on all pull requests #
+######################################
```

Comment:
This comment does not reflect the actual situation.

```suggestion
##########%%%###############
# Lint on each push and PR #
############################
```

Reply:
Fixed. In fact I've rewritten all the comment lines more in keeping with our other actions.